common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -73,8 +76,21 @@
             // Failed to access the address field.
             addressField = null;
         }
+
         logger.debug("java.nio.Buffer.address: {}", addressField != null? "available" : "unavailable");
 
+        Constructor<?> directBufferConstructor;
+        try {
+            directBufferConstructor = direct.getClass().getDeclaredConstructor(long.class, int.class);
+            directBufferConstructor.setAccessible(true);
+        } catch (Throwable t) {
+            directBufferConstructor = null;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1148)